package-lock.json

```diff
@@ -1781,6 +1781,25 @@
       "resolved": "https://registry.npmjs.org/@emotion/unitless/-/unitless-0.7.5.tgz",
       "integrity": "sha512-OWORNpfjMsSSUBVrRBVGECkhWcULOAJz9ZW8uK9qgxD+87M7jHRcvh/A96XXNhXTLmKcoYSQtBEX7lHMO7YRwg=="
     },
+    "node_modules/@endemolshinegroup/cosmiconfig-typescript-loader": {
```

Comment:
I'm always so afraid of `package-lock.json` changes as for me it already feels like I'm constantly dealing with its merge conflicts which are a pain to resolve, and when a PR does not actually upgrade packages we should try to keep those changes out. So can we get this change somehow out of here, like copying over current `package-lock.json` from `feature/multinetwork`?

Reply:
Yup sure I'll copy it from feature `feature/multinetwork,` the lock file was updated when I've bumped the `oceanlib` version to have acces to the `DDO.chainId` but then it was bumped later also on the feature/multinetwork branch so not needed anymore